.env.template

```diff
@@ -1,2 +1,2 @@
-REACT_APP_API_HOST=localhost
+REACT_APP_API_HOST=https://localhost

```

Comment:
Looks good! But I think this should be `http` as on local/dev environment API is served under `http`, but on production, this env variable might starts with `https`.

Update: 
Should be changed in `src/services/authentication.js` too.

Reply:
Thanks!! How about removing .env.template from repo and add .env.development and .env.production files instead?